src/components/user-profile.component.js

```diff
@@ -86,8 +86,8 @@ export const UserProfile = ({
       <ImageZoom
         uri={{
           uri: initialUser.avatar_url
-            ? initialUser.avatar_url
-            : user.avatar_url,
+            ? `${initialUser.avatar_url}&cache=${initialUser.updated_at}`
```

Comment:
Although the param `cache` could be called for example `dateUpdate` (to make it more meaningful), but I would generally convert the date into a timestamp, so it was cleaner, but I think it is superfluous.